resources/services/ssm/patch_baselines.go

```diff
@@ -0,0 +1,302 @@
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "enable_non_security",
+						Description: "For instances identified by the approval rule filters, enables a patch baseline to apply non-security updates available in the specified repository",
+						Type:        schema.TypeBool,
+					},
+					{
+						Name:        "patch_filter_group",
+						Description: "The patch filter group that defines the criteria for the rule.",
+						Type:        schema.TypeJSON,
+						Resolver:    resolveApprovalRulePatchFilterGroup,
+					},
+				},
+			},
+			{
+				Name:          "aws_ssm_patch_baseline_sources",
+				Description:   "Information about the patches to use to update the instances, including target operating systems and source repository",
+				Resolver:      fetchSsmPatchBaselineSources,
+				IgnoreInTests: true,
+				Columns: []schema.Column{
+					{
+						Name:        "patch_baseline_cq_id",
+						Description: "Unique CloudQuery ID of aws_ssm_patch_baselines table (FK)",
+						Type:        schema.TypeUUID,
+						Resolver:    schema.ParentIdResolver,
+					},
+					{
+						Name:        "configuration",
+						Description: "The value of the yum repo configuration",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "name",
+						Description: "The name specified to identify the patch source.  This member is required.",
+						Type:        schema.TypeString,
+					},
+					{
+						Name:        "products",
+						Description: "The specific operating system versions a patch repository applies to, such as \"Ubuntu16.04\", \"AmazonLinux2016.09\", \"RedhatEnterpriseLinux7.2\" or \"Suse12.7\". For lists of supported product values, see PatchFilter.  This member is required.",
+						Type:        schema.TypeStringArray,
+					},
+				},
+			},
+		},
+	}
+}
+
+// ====================================================================================================================
+//                                               Table Resolver Functions
+// ====================================================================================================================
+
+func fetchSsmPatchBaselines(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- interface{}) error {
+	cl := meta.(*client.Client)
+	svc := cl.Services().SSM
+	params := ssm.DescribePatchBaselinesInput{}
+	for {
+		result, err := svc.DescribePatchBaselines(ctx, &params, func(o *ssm.Options) {
+			o.Region = cl.Region
+		})
+		if err != nil {
+			return diag.WrapError(err)
+		}
+		for _, item := range result.BaselineIdentities {
+			b, err := svc.GetPatchBaseline(
+				ctx,
+				&ssm.GetPatchBaselineInput{
+					BaselineId: item.BaselineId,
+				},
+				func(o *ssm.Options) {
+					o.Region = cl.Region
+				},
+			)
+			if err != nil {
+				return diag.WrapError(err)
+			}
+			if b != nil {
+				res <- *b
+			}
+		}
```

Comment:
Same issue of performance and use of go routines

Reply:
Adjusted.